shared/login/relogin/container.tsx

```diff
@@ -42,7 +42,7 @@ const LoginWrapper = (props: Props) => {
 
   const selectedUserChange = React.useCallback(
     user => {
-      dispatch(LoginGen.createLoginError({error: null}))
```

Comment:
this looks a little funky now, when what it's really doing is clearing the error. How would you feel about explicitly passing undefined? (same below)